dj/api/nodes.py

```diff
@@ -493,6 +520,34 @@ def upsert_a_materialization_config(
         if config.engine.name != data.engine.name
     ]
 
+    if current_revision.type in (NodeType.DIMENSION, NodeType.TRANSFORM):
+        materialization_ast = build_node(
+            session=session,
+            node=node.current,
+            filters=(
+                filters_from_partitions(
+                    [
+                        Partition.parse_obj(partition)
+                        for partition in data.config["partitions"]
+                    ],
+                )
+                if data.config["partitions"]
+                else []
+            ),
+            dimensions=[],
+            orderby=[],
+        )
+        generic_config = GenericMaterializationConfig(
+            query=str(materialization_ast),
+            spark=data.config["spark"] if "spark" in data.config else {},
+            partitions=data.config["partitions"] if "partitions" in data.config else [],
+            upstream_tables=[
```

Comment:
Oh, I think I understand. Perhaps we can use this to setup the dependencies for the jobs?

Reply:
Yep, it's mainly so that if the materialization service wants to have dependencies on the upstream tables, it can use this setting.